packages/expo-permissions/src/Permissions.ts

```diff
@@ -37,6 +37,21 @@ export const REMINDERS = 'reminders';
 export const SYSTEM_BRIGHTNESS = 'systemBrightness';
 export const MOTION = 'motion';
 
+// Map corresponding permission to correct package
+const PERMISSION_MODULE_MAPPING = {
+  [CAMERA]: 'expo-camera',
+  [CAMERA_ROLL]: 'expo-media-library',
+  [AUDIO_RECORDING]: 'expo-av',
+  [LOCATION]: 'expo-location',
+  [USER_FACING_NOTIFICATIONS]: 'expo-permissions',
+  [NOTIFICATIONS]: 'expo-permissions',
```

Comment:
We've just moved these requesters to `expo-notifications` 😉 See #8486 

```suggestion
  [USER_FACING_NOTIFICATIONS]: 'expo-notifications',
  [NOTIFICATIONS]: 'expo-notifications',
```